patches/com_grail_bazel_toolchain.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/toolchain/cc_toolchain_config.bzl b/toolchain/cc_toolchain_config.bzl
```

Comment:
I know this patch is just getting copied around, but it really ought to have an upstream PR on it so we could drop in the future. @f0rmiga have you already raised the issue?